pkg/unikontainers/hypervisors/qemu.go

```diff
@@ -80,13 +80,16 @@ func (q *Qemu) Execve(args ExecArgs, ukernel unikernels.Unikernel) error {
 		}
 		netcli += args.TapDevice
 		cmdString += netcli
+	} else {
+		cmdString += " -nic none"
 	}
 	if args.BlockDevice != "" {
 		blockCli := ukernel.MonitorBlockCli(qemuString)
 		if blockCli == "" {
 			blockCli += " -device virtio-blk-pci,id=blk0,drive=hd0,scsi=off"
-			blockCli += " -drive format=raw,if=none,id=hd0,file=" + args.BlockDevice
+			blockCli += " -drive format=raw,if=none,id=hd0,file="
 		}
+		blockCli += args.BlockDevice
```

Comment:
It appears we changed the behavior of this function to always append the args.BlockDevice to blockCli and subsequently to cmdString. Is this change intentional?

Reply:
Yes, if the unikernel does not require any specific argument in Qemu, then we will use the generic one. Also, the unikernel implementation does not need to take any decision based on the file path, hence it is not required (except of just adding it in the string). However, maybe we should document better that the path will be added right after the args.